src/ToDo.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/jsx-no-duplicate-props */
+import React from 'react';
+import style from './Todo.module.scss';
+
+const ToDo = ({
+  toDoList, addToDo, removeToDo, resetTodo,
+}) => {
+  const addInputValue = () => {
+    const input = document.getElementsByClassName(`${style.input}`)[0];
+    if (input.value !== '') {
+      addToDo(input.value);
+    }
+    input.value = '';
+  };
+  const removeToDoItem = (className) => {
+    const shouldRemove = document.getElementsByClassName(`${className}`)[0].innerHTML;
+    removeToDo(shouldRemove);
+  };
+  const toDoArr = [];
+  for (let i = 0; i < toDoList.length; i++) {
+    toDoArr.push(toDoList[i].name);
+  }
```

Comment:
remove it.

all logic must be in actions.
why you use finding elements by selectors?
you don`t need a local array of items. use only **store values.**

don`t use for loop, it will rerender each time of changes.